addons/website_blog/controllers/main.py

```diff
@@ -150,21 +147,18 @@ def _prepare_blog_values(self, blogs, blog=False, date_begin=False, date_end=Fal
         tag_category = tools.lazy(lambda: sorted(all_tags.mapped('category_id'), key=lambda category: category.name.upper()))
         other_tags = tools.lazy(lambda: sorted(all_tags.filtered(lambda x: not x.category_id), key=lambda tag: tag.name.upper()))
         nav_list = tools.lazy(self.nav_list)
-        # for performance prefetch the first post with the others
-        post_ids = (first_post | posts).ids
```

Comment:
Why is this line completely removed? Don't we still need to prefetch some of the posts?

Reply:
nevermind I saw where this was used, there's no need to pass an arg to that method 👍🏻